src/input/input-with-icon.scss

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
+@import '../label/index';
```

Comment:
Why you need this import?

Reply:
It's needed for `.iui-end-icon`.  I could move that into `/input-container/` if you'd rather.
![Screen Shot 2022-05-06 at 1 33 41 PM](https://user-images.githubusercontent.com/849817/167183539-c4783d6a-2ce9-40a8-a33c-1b6ddc4e44e7.png)

Reply:
Ugh, I am not sure. `iui-end-icon` is for `Selelect`, `Combobox` and `input-container` has stuff related with container. Maybe it is ok how it is right now.